packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -196,6 +199,73 @@ router.get(
   })
 );
 
+// Zod schema to validate the request body based on the response of GET /duplicates
+const patchDuplicatesSchema = z.record(
+  // cx
+  z.record(
+    // patient
+    z.record(
+      z
+        .object({
+          // person
+        })
+        .nullish()
+    )
+  )
+);
+
+/** ---------------------------------------------------------------------------
+ * PATCH /internal/patient/duplicates
+ *
+ * Links the patient to the chosen person.
+ * Additionally, unenroll:
+ * - any other person linked to the patient; AND
+ * - all other persons matching the patient's demographics.
```

Comment:
nit "... if the unenrollByDemographics query param is specified"

Good practice for function headers is to describe how to use the route so devs know how to call it at a glance without reading a code